### DISCUSSION TITLE
Billing -> Update Payment Method - Improper Bottom Padding

### DISCUSSION THREAD

**Discussion by vehm** [community/community#43732] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Simply put, there is no bottom padding in the "Billing & plans/ Payment information" box on https://github.com/settings/billing/payment_information.

![improper-padding](https://user-images.githubusercontent.com/83190954/211329072-fad5fd4d-ddb4-4e70-ac75-d79cb117eb0d.png)

The issue is with the box's class: "Box list-style-none pt-3 pr-3 pb 3 pl-3 mb-3"
The bottom padding should be "pb-3" rather than "pb 3".

**Comment by melvincwng** [comment, marked as answer]
Hey @vehm :wave:, thanks for highlighting this issue in GitHub Community.

For bugs, you can actually report them directly to [GitHub Bug Report](https://support.github.com/contact/bug-report).

Hope this clarifies, and have a good day ahead :smile:

**Reply by vehm** [reply]
Thank you for letting me know!

**Comment by vehm** [comment]
Support ticket submitted. Closing this discussion.
